release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -54,6 +54,11 @@ This release adds improvements related to the following components and concepts.
 
 //For more information, see ../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[Preparing to install on IBM Cloud].
 
+[id="ocp-4-10-installation-and-upgrade-vsphere-provisioning"]
+==== Thin provisioning support for VMware vSphere cluster installation
+
+{product-title} {product-version} introduces support for thin provisioned disks when you install a cluster using installer-provisioned infrastructure. You can provision disks as `thin`, `thick`, or `eagerZeroedThick`. For more information on disk provisioning modes in VMware vSphere, see xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html#installation-configuration-parameters_installing-vsphere-installer-provisioned-customizations[Installation configuration parameters].
```

Comment:
I can't find a reference for this, but I've always been taught to use "about" rather than "on" in the phrase "for more information about/on ...." The reasoning was for ESL and translation. "About" is a clearer word than "on" in this context.